### PR TITLE
Add docker build target

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

**Review by zjj2wry — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
-EXECUTABLE=dns-controller-manager
-PROJECT=github.com/gardener/external-dns-management
-VERSION=$(shell cat VERSION)
+REGISTRY              := gcr.io/pingcap-public/gardener
+EXECUTABLE            := dns-controller-manager
+PROJECT               := github.com/gardener/external-dns-management
```
Comment (tennix) [inline Makefile:3]: I'm wondering if this need to be adjusted to the forking repo, but it seems this is not used anywhere.

Comment (aylei) [reply Makefile:3]: Yes, we don't need to update this

**Review by tennix — APPROVED** [review]
LGTM

**Pull request merged**